dnf-behave-tests/dnf/dnf5/download.feature

```diff
@@ -58,3 +58,14 @@ Scenario: Download with alldeps options
    Then stderr contains "Option \"--alldeps\" should be used with \"--resolve\""
     And the exit code is 2
 
+@not.with_dnf=4
```

Comment:
Minor: Could you please add `@dnf5` tag for this whole feature and drop the `@not.with_dnf=4` tag as we are already branched?